api/src/main/java/io/strimzi/api/kafka/model/KafkaResources.java

```diff
@@ -138,6 +138,19 @@ public static String tlsBootstrapAddress(String clusterName) {
         return bootstrapServiceName(clusterName) + ":9093";
     }
 
+    /**
+     * Returns the address (<em>&lt;host&gt;</em>:<em>&lt;port&gt;</em>)
+     * of the generic bootstrap {@code Service} for a {@code Kafka} cluster, and {@code port} of the given.
+     * @param clusterName  The {@code metadata.name} of the {@code Kafka} resource.
+     * @param port The {@code spec.kafka.listeners.port} of the {@code Kafka} resource.
+     * @return The address of the corresponding bootstrap {@code Service}.
+     * @see #plainBootstrapAddress(String)
+     * @see #tlsBootstrapAddress(String)
+     */
+    public static String bootstrapAddressOnSpecificPort(String clusterName, int port) {
+        return bootstrapServiceName(clusterName) + ":" + port;
+    }
+
```

Comment:
I'm not sure I think we really need this here.

Reply:
Ok I will remove it.